tests/Pagerfanta/Tests/Adapter/ConcatenationAdapterTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Pagerfanta\Tests\Adapter;
+
+use Pagerfanta\Adapter\ArrayAdapter;
+use Pagerfanta\Adapter\CallbackAdapter;
+use Pagerfanta\Adapter\ConcatenationAdapter;
+use Pagerfanta\Adapter\FixedAdapter;
+use Pagerfanta\Adapter\NullAdapter;
+
+class ConcatenationAdapterTest extends \PHPUnit_Framework_TestCase
```

Comment:
Since you submitted this PR, we've merged https://github.com/whiteoctober/Pagerfanta/pull/240, which uses namespaced PHPUnit files.  Please could you change this line to be `class ConcatenationAdapterTest extends TestCase` and add the following use statement?

```
use PHPUnit\Framework\TestCase;
```

Reply:
✅